package.json

```diff
@@ -32,18 +32,19 @@
     "package-tarball": "npm run build && npm pack"
   },
   "devDependencies": {
-    "@sketch-hq/sketch-assistant-utils": "6.0.0",
+    "@sketch-hq/sketch-assistant-utils": "^6.4.0",
     "@types/jest": "26.0.4",
     "@types/node": "14.0.22",
-    "jest": "26.6.3",
+    "jest": "^27.0.3",
     "node-fetch": "2.6.1",
     "prettier": "2.0.5",
-    "ts-jest": "26.1.2",
-    "typescript": "3.9.6",
+    "ts-jest": "^27.0.2",
+    "typescript": "^4.3.2",
     "webpack": "4.43.0",
     "webpack-cli": "3.3.12"
```

Comment:
Is there any particular reason not to update these and the other ones above? We recently made the jump in [@sketch-hq/sketch-assistants](https://github.com/sketch-hq/sketch-assistants/blob/986cddcc8d9ae2ab357a9f41bd5593ee6a4d8e2a/package.json#L20-L21), too.

Reply:
Just wanted to avoid changing too many bits at once, but I’ll do them too